docs/administration/robot-accounts/_index.md

```diff
@@ -52,15 +40,16 @@ This page lists all available system robot accounts for your Harbor instance. Th
 1. Go to **Administration**, select a project, and select **Robot Accounts**.
 1. Click **New Robot Account**.
```

Comment:
1. Log in to the Harbor user interface, with system administrator privileges.
1. Go to **Administration**, select a project, and select **Robot Accounts**.
1. Click **New Robot Account**.